apps/accelerate/speedster/docs/en/docs/installation.md

```diff
@@ -30,7 +30,59 @@ Great, now you are now ready to accelerate your model 🚀 Please visit the foll
+- `--compilers` is used to specify the deep learning compilers to be installed in addition to the frameworks. The supported compilers are: `deepsparse`, `tensor_rt`, `torch_tensor_rt`, `onnxruntime`, `openvino`and `intel_neural_compressor`. You can specify multiple compilers by separating them with a space. For example, `--compilers deepsparse tensor_rt` will install DeepSparse and TensorRT. If you want to install all compilers, you can use `all` as the argument. For example, `--compilers all` will install all compilers. Speedster supports also `torchscript` and `tf_lite` compilers, but they are pre-installed inside the frameworks, so you don't have to include them in this list. Speedster also supports `tvm`, which is not currently supported by the auto-installer and must be installed manually, see the next section if you want to include it.  Default: `all`.
+
+Let's see an example of how to use these arguments:
+
+    python -m nebullvm.installers.auto_installer --frameworks torch --extra-backends all --compilers all
+
+This command will install PyTorch, as well as all PyTorch supported backends and compilers.
+
+The following table shows the supported combinations of frameworks, backends and compilers that you can install with the auto-installer:
+
+| Framework   | Extra Backends            | Compilers                                                                                        |
+|-------------|---------------------------|--------------------------------------------------------------------------------------------------|
+| PyTorch       | ONNX                      | DeepSparse, TensorRT, Torch TensorRT, ONNXRuntime, OpenVINO, Intel Neural Compressor |
+| TensorFlow  | ONNX                      | ONNXRuntime, TensorRT, OpenVINO                                                      |
+| ONNX        | /                         | ONNXRuntime, TensorRT, OpenVINO                                                     |
+| HuggingFace | PyTorch, TensorFlow, ONNX | DeepSparse, TensorRT, Torch TensorRT, ONNXRuntime, OpenVINO, Intel Neural Compressor           |
+
+### Manual installation
+
+If you want to install the requirements manually, this section collects the links to the official installation guides of all the frameworks and compilers supported by `Speedster`.
```

Comment:
What about replace this sentence with:

If you want to manually install the requirements, this section collects links to the official installation guides for all frameworks and compilers supported by `Speedster`.